ocean_lib/web3_internal/constants.py

```diff
@@ -2,6 +2,22 @@
 # Copyright 2021 Ocean Protocol Foundation
 # SPDX-License-Identifier: Apache-2.0
 #
+
+'''
+This module holds following default values:
+
+`ENV_GAS_PRICE` = "GAS_PRICE"
+
+`ENV_MAX_GAS_PRICE` = "MAX_GAS_PRICE"
+
+`GAS_LIMIT_DEFAULT` = 1000000
+
+`MIN_GAS_PRICE` = 1000000000
+
+`ZERO_ADDRESS` = "0x0000000000000000000000000000000000000000"
```

Comment:
DRY Principle: Don't Repeat Yourself.

A key mantra in software dev: don't violate DRY. It hurts maintainability, causes brittle code.

This change violates DRY.

To fix, simply change the docs part to saying something like: "This module holds default values for gas price, gas limit, and more"